File: tasks-api-java/src/main/java/br/com/darlan/tasks/model/Task.java
package br.com.darlan.tasks.model;

import jakarta.persistence.*;
import lombok.AllArgsConstructor;
import lombok.Builder;
import lombok.Data;
import lombok.NoArgsConstructor;

import java.io.Serializable;
import java.time.LocalDate;
import java.time.LocalTime;

@Entity
@Table(name = "TB_TASKS")
@Data
@Builder
@AllArgsConstructor
@NoArgsConstructor
public class Task implements Serializable {
    @Id @GeneratedValue(strategy = GenerationType.IDENTITY)
    Long idTask;
    String taskName;
    String taskDescription;
    Boolean uniqueExecution;
    LocalDate startDate;
    LocalTime startTime;
    LocalDate endDate;
    LocalTime endTime;
    String address;
    String host;
}

File: kotlin/bff-tasks-kotlin/src/main/kotlin/br/com/darlan/bfftaskskotlin/model/Task.kt
package br.com.darlan.bfftaskskotlin.model

import java.time.LocalDate
import java.time.LocalTime

class Task {
    var idTask: Long? = null;
    var taskName: String? = null;
    var taskDescription: String? = null;
    var uniqueExecution: Boolean? = null;
    var startDate: LocalDate? = null;
    var startTime: LocalTime? = null;
    var endDate: LocalDate? = null;
    var endTime: LocalTime? = null;
    var address: String? = null;
    var host: String? = null;
}